frontend/packages/pipelines-plugin/locales/en/pipelines-plugin.json

```
@@ -196,6 +196,7 @@
   "Select a Project to view the list of Pipelines": "Select a Project to view the list of Pipelines",
   "Create Pipeline": "Create Pipeline",
   "by name": "by name",
+  "{{resource}} Results": "{{resource}} Results",
```

```suggestion
  "{{resourceName}} Results": "{{resourceName}} Results",
```
Suggest upping the variable name used here as the translators could use it to help them understand context.